langchain4j-mistral-ai/src/main/java/dev/langchain4j/model/mistralai/MistralAiClient.java

```diff
@@ -125,9 +135,17 @@ public void onEvent(EventSource eventSource, String id, String type, String data
                     try {
                         MistralAiChatCompletionResponse chatCompletionResponse = GSON.fromJson(data, MistralAiChatCompletionResponse.class);
                         MistralAiChatCompletionChoice choice = chatCompletionResponse.getChoices().get(0);
+
                         String chunk = choice.getDelta().getContent();
-                        contentBuilder.append(chunk);
-                        handler.onNext(chunk);
+                        if (chunk != null){
```

Comment:
minor: 
```suggestion
                        if (isNotNullOrEmpty(chunk)){
```
Sometimes APIs produce empty strings (""). In such case I guess it makes no sense to call the handler